README.md

```diff
@@ -28,3 +28,36 @@ mounting of the file-system implementation may be available for all implementati
 ## Documentation
 
 Please refer to [RTD](https://filesystem-spec.readthedocs.io/en/latest/?badge=latest)
+
+## Develop
+
+### Code Formatting
+
+Filesystem_spec uses [Black](https://black.readthedocs.io/en/stable) to ensure
+a consistent code format throughout the project. ``black`` can be installed
+with:
+
+```
+pip install black
+```
+
+and then run from the root of the filesystem_spec repository:
+
+```
+black fsspec
+```
+
+to auto-format your code. Additionally, many editors have plugins that will
+apply `black` as you edit files.
+
+Optionally, you may wish to setup [pre-commit hooks](https://pre-commit.com)
+to automatically run `black` and ``flake8`` when you make a git commit. This
+can be done by installing `pre-commit` from the repository root:
+
+```
+pip install pre-commit
```

Comment:
(also on conda)
Note that you mention flake8 but don't include it in the instructions.